parsers/to-style-dictionary/README.md

```diff
@@ -0,0 +1,460 @@
+| `formatTokens.colorFormat.format`  | optional | `rgb` `prgb` `hex` `hex6` `hex3` `hex4` `hex8` `name` `hsl` `hsv` | `hex`       | The color format you want to apply to your potential color design token.                                                                                                                                   |
+| `formatTokens.fontSizeFormat.unit` | optional | `px` `rem`                                                        | `none`      |                                                                                                                                                                                                            |
+| `formatTokens.fontFormat`          | optional | `woff2` `woff` `otf` `ttf` `eot`                                  | `ttf`       | The formats of font files you use.                                                                                                                                                                         |
+| `splitBy`                          | optional | `string`                                                          |             | The character used to define the nesting of the values in the object (e.g. The name of the color in [this example](https://github.com/Specifyapp/parsers/tree/master/parsers/to-style-dictionary#input-2)) |
+| `assetsBaseDirectory.fonts`        | optional | `string`                                                          | `none`      | The base directory containing your font files.                                                                                                                                                             |
+
+|
+| `assetsBaseDirectory.images` | optional | `string` | `none` | The base directory containing your images.
+|
+| `assetsBaseDirectory.icons` | optional | `string` | `none` | The base directory containing your icons. |
+| `formatConfig.endOfLine` | optional | `auto` `lf` `crlf` `cr` | `auto` | [Prettier documentation](https://prettier.io/docs/en/options.html#end-of-line) |
+| `formatConfig.tabWidth` | optional | `number` | `2` | [Prettier documentation](https://prettier.io/docs/en/options.html#tab-width) |
+| `formatConfig.useTabs` | optional | `boolean` | `true` | [Prettier documentation](https://prettier.io/docs/en/options.html#tabs) |
+
+## Types
+
+ℹ️ **Please be aware that, depending on the order you use parsers, their input and output types have to match.**
+
+### Input
+
+Array of object with at least `name`, `value` and `type`:
+
+```ts
+Array<{name: string, value: any, type: string}>
+```
+
+### Output
+
+An array of object containing a `name` and a `value`. The value is an object containing either an `url` or a `content`. This object is considered as a `DownloadableFile`.
```

Comment:
An array of objects containing a `name` and a `value`. The value is an object containing either an `url` or a `content`. This object is considered as a `DownloadableFile`.

Reply:
I don't see the difference here with what is currently written